pages/dkp/kommander/1.3/clusters/index.md

```diff
@@ -181,8 +181,8 @@ data:
 ```
 
 | Key | Description | Required |
-| :--- | :--- | :---: | 
-| metadata . labels . "d2iq.io/addon" | The addon name (id) | X |
+| :--- | :--- | :---: |
+| metadata . labels . "d2iq.io/addon" | The platform service name (id) | X |
```

Comment:
@gracedo where we say "platform service name", is that referencing the product UI?